cli/npm/resolution/graph.rs

```diff
@@ -159,6 +159,9 @@ impl ResolvedNodeIds {
   }
 }
 
+// todo(dsherret): for some reason the lsp errors when using an Rc<RefCell<T>> here.
+// We should investigate and fix.
```

Comment:
Why do you want to use `Rc<RefCell<T>>`?

Reply:
I'll update the comment to be better. It's instead of Arc<Mutex<NodeId>>, because this code isn't multi-threaded.